scanners/trustymail.py

```diff
@@ -64,7 +60,8 @@ def init_domain(domain, environment, options):
         # http://www.dnspython.org/docs/1.14.0/dns.resolver-pysrc.html#Resolver._compute_timeout.
         resolver.timeout = float(timeout)
         resolver.lifetime = float(timeout)
-        resolver.nameservers = dns_hostnames
+        if dns_hostnames is not None:
```

Comment:
What happens if it *IS* `None` ?

Should there be a fallback / default value still?

Reply:
If it is `None` then the system DNS servers as defined in `/etc/resolv.conf` will be used.